CMakeLists.txt

```diff
@@ -453,7 +453,7 @@ endif ()
 set (CMAKE_POSTFIX_VARIABLE "CMAKE_${CMAKE_BUILD_TYPE_UC}_POSTFIX")
 
 set (CMAKE_POSITION_INDEPENDENT_CODE OFF)
-if (OS_LINUX AND NOT ARCH_AARCH64)
+if (OS_LINUX AND NOT ARCH_AARCH64 AND NOT ARCH_S390X)
```

Comment:
I am rather confused by l. 456-462. Before your change, PIE was disabled for x86, PPC and s390x (i.e. `NOT ARCH_AARCH64`, l. 456). But the comment says PIE is "disabled" on ARM - actually ARM is the only platform where PIE is enabled. While at it, would you like to change the wording of the comment here? Thanks.

Please also add a comment along the lines of l. 458 explaining *why* PIE is disabled as well on s390x.